engine/src/flutter/shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -1649,6 +1649,100 @@ TEST_F(EmbedderTest, CanRemoveView) {
   ASSERT_EQ(message, "View IDs: [0]");
 }
 
+TEST_F(EmbedderTest, RemoveViewCallbackIsInvokedAfterRasterThreadIsDone) {
+  auto& context = GetEmbedderContext<EmbedderTestContextSoftware>();
+  EmbedderConfigBuilder builder(context);
+  std::mutex engine_mutex;
+  UniqueEngine engine;
+  auto render_thread = CreateNewThread("custom_render_thread");
```

Comment:
Is this mimicking the UI thread? Should we call it the UI thread instead?

Reply:
It's mimicking raster thread:
```cpp
builder.SetRenderTaskRunner(
      &render_task_runner.GetFlutterTaskRunnerDescription());
```